manifests/rbac.yaml

```diff
@@ -0,0 +1,55 @@
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
```

Comment:
Should have some standard/common labels as well.

Reply:
Captured in https://github.com/kubernetes-sigs/container-object-storage-interface-controller/issues/23